tableauserverclient/server/endpoint/groups_endpoint.py

```diff
@@ -1,5 +1,6 @@
 from .endpoint import Endpoint
 from .exceptions import MissingRequiredFieldError
+from ...models.exceptions import UnpopulatedPropertyError
```

Comment:
... is the same as ../.. on the command line.  I know it's a bit annoying, but we need relative imports.